sopel/modules/translate.py

```diff
@@ -70,7 +70,7 @@ def translate(text, in_lang='auto', out_lang='en'):
         data = json.loads(result)
     except ValueError:
         LOGGER.error(
-            'Error parsing JSON response from translate API (%s to %s: "%s")',
+            "Error parsing JSON response from translate API (%s to %s: '%s')",
```

Comment:
Why?

Reply:
Oh, that might have been either an honest mistake (I did some find & replace regex), or there is something else. I think it's probably a mistake.